test/integration/smart_answer_flows/business_coronavirus_support_finder_test.rb

```diff
@@ -1,58 +0,0 @@
-require_relative "../../test_helper"
```

Comment:
I understand why this needs deleting - but it doesn't really relate to this PR.  Should it be a separate PR?

Reply:
Yup - so because I had changed the flow I'd need to update these tests to make them pass. Figured if they are going to be deleted anyways thats the easier change to make.

Reply:
No worries